stan/math/prim/scal/fun/lgamma.hpp

```diff
@@ -29,21 +44,47 @@ namespace math {
    \end{cases}
 \f]
 *
+* Note: Please refer to https://github.com/stan-dev/math/pull/1255 for
```

Comment:
I don’t understand this comment. What is the actual requirement you’re trying to spell out? Does it not fit here?